### ISSUE TITLE
Any plans to add preview / open files without actually downloading them? 

### ISSUE THREAD

**Issue by pixincreate** [FilenCloudDienste/filen-mobile#35]
It's really annoying to see files getting downloaded instead of allowing user to view directly within the app/browser.
Everytime, a user wants to view a video without actually downloading, it forcefully downloads, leaving no option behind.

An example for such a feature is GDrive. You can view photos and videos without downloading. They load and buffer then and there itself.

**Comment by DanielProg39** [comment]
I fully agree with you. This was actually possible in an old version of app

**Comment by pixincreate** [comment]
I'm unsure which version. App version directly jimoed from 1.0.53 to 2.0.12 and both the versions didn't had the feature AFAIK. This doesn't exist even when filen_io is accessed from the web directly.

Yes, some of them *used* to load but most of them asked me to download.

**Comment by Dwynr** [comment]
Working on improving the previews.

**Comment by MTH607** [comment]
It would be really beneficial if especially in the mobile app, you could see a preview of the video without downloading it.

**Comment by hruzgar** [comment]
Yes i also want to chime in on this. Looking at [s3drive](https://s3drive.app/), streaming encrypted video is definitely possible. Instead of downloading the full video, you just have to download some specified size and amount of chunks (depending on the timestamp of the video) and decrypt them on the fly. The math really isn't that hard i think. Would be so dope if we had this not only in the app itself but also in the web version!
